/***************************************************************************
    *   Copyright (C) 2016 by Renaud Guezennec                                *
    *   http://www.rolisteam.org/contact                                      *
    *                                                                         *
    *   rolisteam is free software; you can redistribute it and/or modify     *
    *   it under the terms of the GNU General Public License as published by  *
    *   the Free Software Foundation; either version 2 of the License, or     *
    *   (at your option) any later version.                                   *
    *                                                                         *
    *   This program is distributed in the hope that it will be useful,       *
    *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
    *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
    *   GNU General Public License for more details.                          *
    *                                                                         *
    *   You should have received a copy of the GNU General Public License     *
    *   along with this program; if not, write to the                         *
    *   Free Software Foundation, Inc.,                                       *
    *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
    ***************************************************************************/

#ifndef PDFMANAGER_H
#define PDFMANAGER_H

#include <QDialog>

namespace Ui {
class PdfManager;
}

class PdfManager : public QDialog
{
    Q_OBJECT

public:
    explicit PdfManager(QWidget *parent = 0);
    virtual ~PdfManager();

    qreal getDpi();

    bool hasResolution();
    int getWidth();
    int getHeight();

    void setHeight(int h);
    void setWidth(int w);

signals:
    void resolutionChanged();
    void apply();

private:
    Ui::PdfManager *ui;
};

#endif // PDFMANAGER_H
